manifests/s/Swift/Toolchain/5.5.3/Swift.Toolchain.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=NVS1.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.5.3
```

Comment:
```suggestion
PackageVersion: 0.0.0.0
```

Reply:
May I ask what exactly this does? I assume it's a workaround, and not the value we actually want here?